app/components/Switch.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 
-export const Switch = function(props = {isOn: true, handleSwitch: () => {}}) {
+export const Switch = (props = { isOn: true, handleSwitch: () => {} }) => {
```

Comment:
Don't default this way. Use `defaultProps`. Also no need to default `handleSwitch` with empty method here.
make it as required method.